php7.2.Dockerfile

```diff
@@ -6,6 +6,10 @@ ENV EVAL=true
 
 RUN BRANCH="$BRANCH" /initnc.sh
 
+RUN apt-get update && \
+    apt-get install -y composer && \
+    apt-get autoremove -y && apt-get autoclean && apt-get clean && \
+    rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*
```

Comment:
This does not work.
I'll try to make it work.

Reply:
It works in the 7.4  image. For the 7.2 images we should probably just install the apps from the appstore as discussed in #9